Core/Src/pdu.c

```diff
@@ -98,6 +139,61 @@ pdu_t *init_pdu(I2C_HandleTypeDef *hi2c)
 
 	pdu->hi2c = hi2c;
 
+	/* Initialize Motor Controller Current Sensor */
```

Comment:
Can you add one comment above saying these calibration constants were retrieved from altium and give the date?  Just in case they change